src/Aspire.Hosting.Azure.EventHubs/AzureEventHubsResource.cs

```diff
@@ -23,9 +25,18 @@ public class AzureEventHubsResource(string name, Action<ResourceModuleConstruct>
     /// </summary>
     public BicepOutputReference EventHubsEndpoint => new("eventHubsEndpoint", this);
 
+    internal EndpointReference EmulatorEndpoint => new(this, "emulator");
+
+    /// <summary>
+    /// Gets a value indicating whether the Azure Event Hubs resource is running in the local emulator.
+    /// </summary>
+    public bool IsEmulator => this.IsContainer();
+
     /// <summary>
     /// Gets the connection string template for the manifest for the Azure Event Hubs endpoint.
     /// </summary>
     public ReferenceExpression ConnectionStringExpression =>
-        ReferenceExpression.Create($"{EventHubsEndpoint}");
+        IsEmulator
+        ? ReferenceExpression.Create($"Endpoint=sb://{EmulatorEndpoint.Host}:{EmulatorEndpoint.Property(EndpointProperty.Port)};SharedAccessKeyName=RootManageSharedAccessKey;SharedAccessKey=SAS_KEY_VALUE;UseDevelopmentEmulator=true;EntityPath={Hubs.First().Name}")
```

Comment:
Hubs.First() seems problematic

Reply:
I think we might not require the `EntityPath` actually if it's provided in the settings, so I will try to remove it.

Reply:
Done